ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,6 @@
+Make sure you try the following before submitting the issue, thank you!
+
+- Check that umake is the latest version (16.11.1)
```

Comment:
It seemed more convenient and harder to ignore if the correct version is there. But sure, I can remove it if you think it's better. There is the problem that at the moment the github releases does not show the latest.